Tests/ClientTests/Frontend/Home/FirefoxHomeViewModelTests.swift

```diff
@@ -35,9 +35,9 @@ class FirefoxHomeViewModelTests: XCTestCase {
                                           tabManager: MockTabManager(),
                                           urlBar: URLBarView(profile: profile),
                                           referrals: .init())
-        XCTAssertEqual(viewModel.shownSections.count, 5) //Ecosia: Update number of sections
-        XCTAssertEqual(viewModel.shownSections[0], HomepageSectionType.logoHeader)
-        XCTAssertEqual(viewModel.shownSections[1], HomepageSectionType.libraryShortcuts)
-        XCTAssertEqual(viewModel.shownSections[2], HomepageSectionType.impact)
+        XCTAssertEqual(viewModel.shownSections.count, 6) //Ecosia: Update number of sections
+        XCTAssertEqual(viewModel.shownSections[0], HomepageSectionType.logoHeader) //Ecosia: Update due to new Ecosia HomePage
+        XCTAssertEqual(viewModel.shownSections[1], HomepageSectionType.bookmarkNudge) //Ecosia: Update due to new Ecosia HomePage
+        XCTAssertEqual(viewModel.shownSections[2], HomepageSectionType.libraryShortcuts) //Ecosia: Update due to new Ecosia HomePage
```

Comment:
Strange that this is not failing on main ever since the NTP changes if those are indeed required. Any idea why?

Reply:
💯 agree for them not failing on `main`.
The only thing I can think of is the region set to the simulator letting appear different sections? If that's the root cause, will these tests be reliable given our current feature-flagged-based configuration?

Reply:
Reverted. Seems like the CI is happy now. Wonder whether the point raised above could be valid ☝️

Reply:
Awesome that I had to revert that eventually. As it is less FF code modified 👌